### PR TITLE
Minor performance improvement

### PR DESCRIPTION
**Pull request by GuhDoy**
I think this would improve performance at some points while not significantly increasing the complexity of the code. Could you please review my PR? Thanks!

### CODE REVIEW

**Comment by GuhDoy** [comment]
Well, I suddenly come up with another idea, which is more capable and simpler.

File: lib/src/main/java/my/nanihadesuka/compose/LazyColumnScrollbar.kt [hunk 1]
```diff
@@ -226,8 +227,8 @@ fun InternalLazyColumnScrollbar(
             val offset = realFirstVisibleItem
                 ?.size
                 ?.let { it.toFloat() * remainder }
-                ?.toInt() ?: 0
-            listState.scrollToItem(index = index, scrollOffset = offset)
+                ?: 0f
+            listState.scrollBy(offset)
```
Comment (nanihadesuka) [inline lib/src/main/java/my/nanihadesuka/compose/LazyColumnScrollbar.kt:231]: Although it works, there is a good reason not to use relative displacement (scrollBy) but instead use absolute displacement (scrollToItem). When the user holds the scrollbar and moves it the "micro-displacements" could cause instability (numerical) if done by relative displacement as the previous calculations might not have been done (recomposition) or skipped as a new one are created (remember we are using a coroutine). Better to have deterministic behaviour.

Note that this doesn't show in the UI unit tests (I don't know how to reproduce programmatically).

**Comment by nanihadesuka** [comment]
@GuhDoy thanks for trying improving it but I think current implementation has better UI stability. 

Will keep open for a week in case you find a good counter argument 😄

**Comment by GuhDoy** [comment]
I don't think the recomposition you mentioned is possible. I wrote the following shit code to detect the recomposition behavior you mentioned:
```
        coroutineScope.launch {
            listState.scrollToItem(index = index, scrollOffset = 0)
            val offset = realFirstVisibleItem
                ?.size
                ?.let { it.toFloat() * remainder }
                ?: 0f
            Thread.sleep(100) // Sleep for 0.1 seconds to pretend we're doing something costly.
            val realFirstVisibleItem2 = listState.layoutInfo.visibleItemsInfo.firstOrNull {
                it.index == index
            }
            if (realFirstVisibleItem2 !== realFirstVisibleItem || realFirstVisibleItem2!!.offset != 0) {
                Log.e("TAG", "recomposition") // <-- Log goes here!
            }
            listState.scrollBy(offset)
        }
```
The log will only be printed when scrolling to the bottom of the `LazyColumn`.

**Comment by nanihadesuka** [comment]
@GuhDoy ok, tested in real device. Seems there are no apparent issues and test pass. Accepted

**Review by nanihadesuka — APPROVED** [review]

**Pull request merged**